app/models/embeddable_host.rb

```diff
@@ -53,6 +54,13 @@ def self.url_allowed?(url)
 
   private
 
+  def reset_embedding_settings
+    unless EmbeddableHost.any?
+      settings = Embedding.settings.map(&:to_s)
+      settings.map {|s| SiteSetting.set(s, SiteSetting.defaults[s.to_sym])}
```

Comment:
I think this can be a one-line 😉 

```suggestion
      Embedding.settings.each { |s| SiteSetting.set(s.to_s, SiteSetting.defaults[s]) }
```